partner_view_360/models/res_partner.py

```diff
@@ -23,6 +23,7 @@
 import logging
 import threading
 import base64
+from datetime import datetime
```

Comment:
consider changing this and next line to:

`from datetime import datetime, date
`